### PR TITLE
rewrote \skillsection{SRU}

### PR DESCRIPTION
**Pull request by hvwaldow**
An attempt to make that paragraph more suited for *4.2. Research Skills*.

+ Starting that paragraph with "FAIR" somehow primes (at least me) for
undestanding "re-use" in the "production sense".

+ I tried to focus this on skills that are *not* necessary for re-use in
traditional SE.

+ The climate model example I was mentioning in the meeting is mentioned here:
https://agupubs.onlinelibrary.wiley.com/doi/full/10.1029/2022MS003588 but I
guess that is too esoteric to mention it in the SRU paragraph.

+ The bit about "software citation and metadata skills" is redundant because
that is also mentioned in \skillsection{SP}.

### CODE REVIEW

**Comment by jpthiele** [comment]
We may have to think about whether some of the citation (if not all) should be moved from SP to SRU
instead of removing it here.
The technical part about providing rich metadata should possibly stay with SP,
but I think everything else could move here.
Especially, since one of the sentences has the word `reuse` in there directly.

**Comment by jpthiele** [comment]
Then we might just call the other section Software Publication being concerned with the technical site of the 
developer and re-use being more the user perspective.

**Comment by CaptainSifff** [comment]
For me the software discovery aspect might be lost....?

File: competencies.md [hunk 1]
```diff
@@ -510,13 +510,16 @@ good research, including publications, reviews and reproducibility.
...
+The re-use of existing assets such as libraries and pieces of code
+to improve efficiency and quality
+belongs to the fundamentals of software construction [@swebok_2014].
+To evaluate whether the artifacts to be re-used suit their needs,
+RSEs often need to consider the scientific context of their origin.
+For example, a paper that references the code under consideration
+might be crucial to validate its fitness for purpose or to discredit it.
+Code that incorporates research-domain specific knowledge
+needs to be understood at a very detailed level
+and its re-use documented to meet standards of good research practice.
```
Comment (MakisH) [inline competencies.md:522]: I tried to emphasize a bit more (even more) the relation to research, focusing on the evaluation of the models and numerical methods. This is something that probably distinguishes a specialized RSE from a "generic" software engineer.

```suggestion
and its re-use documented to meet standards of good research practice.
Not only the technical compatibility needs to be understood and documented
(programming languages, system interoperability), but also the underlying
models and computational methods needs to fit the purpose; this question
often requires wider research skills and deeper understanding of the
research domain at hand.
```

**Review by MakisH — COMMENTED** [review]
I understand that in this section we really want to focus on "getting things off-the-shelf, instead of reinventing the wheel all the time", without addressing how to **make** software reusable. Is that true?

Or at least this is how I read both the old and the new version.

Already addresses more explicitly the relation to research. I added another sentence that I hope helps.

File: competencies.md (hunk 1 above)
Comment (mhagdorn) [reply competencies.md:522]: I think this is a really good addition.  I do wonder if that is one of the big differences between an RSE and a "normal" SE, having to deal with very different fitness-for-purpose metrics. 

BTW, on line 525, drop the s in need

**Pull request merged**